src/parser/priest/shared/modules/shadowlands/covenants/Mindgames.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+
+import ItemHealingDone from 'interface/ItemHealingDone';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+
+import SPELLS from 'common/SPELLS';
+import { formatNumber } from 'common/format';
+import COVENANTS from 'game/shadowlands/COVENANTS';
+
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { AbsorbedEvent, DamageEvent, EnergizeEvent, HealEvent } from 'parser/core/Events';
+import { Options } from 'parser/core/Module';
+
+import AtonementDamageSource from 'parser/priest/discipline/modules/features/AtonementDamageSource';
+import isAtonement from 'parser/priest/discipline/modules/core/isAtonement';
+import SPECS from 'game/SPECS';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+import ItemInsanityGained from 'parser/priest/shadow/interface/ItemInsanityGained';
+import ItemDamageDone from 'interface/ItemDamageDone';
+import ItemManaGained from 'interface/ItemManaGained';
+
+// Shadow: https://www.warcraftlogs.com/reports/Bx7h3bzGKm9CHqF6#fight=1&type=damage-done&source=10
+// Holy: https://www.warcraftlogs.com/reports/MCyn6jhQf13YbRHt#fight=14&type=healing&source=41
+// Disc: https://www.warcraftlogs.com/reports/NWctPky1vKapJVM8#fight=10&type=healing&graphperf=1&source=183
+class Mindgames extends Analyzer {
+  directHealing = 0;
+  preventedDamage = 0;
+  totalDamage = 0;
+  manaGenerated = 0;
+
+  // Disc Specific
+  atonementDamageSource: AtonementDamageSource | null = null;
+  atonementHealing = 0;
+
+  // Shadow Specific
+  insanityGenerated = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasCovenant(COVENANTS.VENTHYR.id);
+
+    if(this.selectedCombatant.spec === SPECS.DISCIPLINE_PRIEST) {
+      this.atonementDamageSource = this.owner.getModule(AtonementDamageSource);
```

Comment:
This is conditionally loaded so that all three priest specs can share one module. If anyone has string feelings against this, I'm happy to split disc into it's own module.

Reply:
No that is perfectly fine way to do it